benchmarks/nel/scripts/datasets/dataset.py

```diff
@@ -214,52 +254,68 @@ def evaluate(
                 for doc in DocBin().from_disk(test_set_path).get_docs(self._nlp_best.vocab)
             ]
         self._nlp_best.config["incl_prior"] = False
+        # Set up spacyfishing.
+        self._nlp_base.add_pipe("entityfishing", last=True)
```

Comment:
This should be done conditionally based on whether `self.spacyfishing == True`, right?